npc/jobs/1-1e/taekwon.txt

```diff
@@ -32,10 +32,10 @@
 //================= Description ===========================================
 //= Official iRO TaeKwon Job Change Quest
 //================= Current Version =======================================
-//= 1.4
+//= 1.5
```

Comment:
I'd go with @kisuka 's idea (not sure if it's written anywhere yet, but it was already discussed once).

Version numbers made of three numbers: major.minor.patch

Script rewrite -> Changes the major number
Important changes -> changes the minor number
Small fixes -> changes the patch number

Not sure about this specific file, since it's not really a trivial fix (it changes a NPC's position), so it might deserve a 1.4->1.5 version bump.
A change like the one below, in `npc/merchants/refine.txt` is just a sprite change and a facing direction change, so it would be enough to bump it from 3.3 to 3.3.1 (not 3.5 in any case o.o)

Calling @kisuka 's opinion on this